content/src/content/jcr_root/apps/core/wcm/components/image/v3/image/clientlibs/editor/js/image.js

```diff
@@ -143,10 +143,25 @@
 
         }
 
-        $(window).adaptTo("foundation-registry").register("foundation.validation.selector", {
-            submittable: ".cmp-image__editor-alt-text",
-            candidate: ".cmp-image__editor-alt-text:not(:hidden)",
-            exclusion: ".cmp-image__editor-alt-text *"
+        $(window).adaptTo("foundation-registry").register("foundation.validation.validator", {
```

Comment:
Wondering if we could move this validator to a commons script and reuse it between the two versions.

Reply:
Indeed, makes sense. Please check 2c1016387d4002473a9e72270b77cae460ef8367